src/azure-cli/azure/cli/command_modules/vm/commands.py

```diff
@@ -303,6 +303,11 @@ def load_command_table(self, _):
         g.custom_command('show-runs', 'show_build_output')
         g.command('cancel', 'begin_cancel')
 
+    with self.command_group('image builder identity', image_builder_image_templates_sdk, custom_command_type=image_builder_custom) as g:
+        g.custom_command('assign', 'assign_template_identity', supports_local_cache=True)
+        g.custom_command('remove', 'remove_template_identity', supports_local_cache=True)
```

Comment:
Do we need to consider add `confirmation=True` for the remove command? @feiyushi

Reply:
`-y` ? I personally think it doesn't hurt to have. also please refer to other command groups if they have it...

Reply:
OK, since similar `disk-encryption-set identity remove` command also have the confirmation step, so we also add this